en/drafts/originals/interactive-data-visualization-dashboard.md

```diff
@@ -22,110 +22,115 @@ doi: XX.XXXXX/phen0000
+
+### Lesson Goals
+
+In this lesson, you will learn how to use Python to:
   * Retrieve data using an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API)
-  * Create the dashboard frontend that determines how it looks
-  * Create the dashboard backend that determines how users interact with it
+  * Create the dashboard frontend (how it looks)
+  * Create the dashboard backend (how users interact with it)
   * Deploy the dashboard onto the web for free
 
-Other essential steps such as installing necessary libraries, setting up a [virtual environment](https://docs.python.org/3/library/venv.html#venv-def), and manipulating the downloaded data will be included when appropriate as well. The code to be executed in the command line will start with the symbol `$`.
+We'll also work through other essential steps such as installing necessary libraries, setting up a [virtual environment](https://docs.python.org/3/library/venv.html#venv-def), and manipulating the downloaded data. The code to be executed in the command line will always start with the symbol `$`.
 
-# Case Study
-The case study concerns how U.S. television stations have covered the current war between Russia and Ukraine. One can compare whether the stations have mentioned the keywords related to Ukraine as frequently as the keywords related to Russia. Further, we can also compare the coverage frequency among some major stations. 
+### Case Study
 
-Quantitative methods for content analysis (CA) have long been a tradition in mass communication studies, and the method of algorithmic text analysis (ATA) has become popular given the rising availability of large amounts of textual data.[^2] Both methods aim to infer meanings from text through classification or measurement. Whereas CA relies heavily on a carefully crafted codebook based on research questions and multiple human coders to ensure the reliability and validity of a systematic analysis,[^3] [^4] ATA relies on algorithms and models (a more general term for this method is [text mining](https://en.wikipedia.org/wiki/Text_mining) or [natural language processing](https://en.wikipedia.org/wiki/Natural_language_processing)).[^5] 
+The case study investigates how U.S. television stations have covered the current [war between Russia and Ukraine](https://en.wikipedia.org/wiki/Russo-Ukrainian_War). For example, we could compare the frequency of Ukraine-related keywords to Russia-related keywords employed by different stations. We could also compare the amount of coverage of the war between certain stations. 
```

Comment:
I've added this wikipedia link to give context to our readers, but would you prefer the following article instead? https://en.wikipedia.org/wiki/Russian_invasion_of_Ukraine

Reply:
Thanks! The current link works better.